Library/TeamTalkLib/teamtalk/Common.h

```diff
@@ -219,13 +221,13 @@ namespace teamtalk {
                             USERRIGHT_TRANSMIT_VIDEOCAPTURE |
                             USERRIGHT_TRANSMIT_DESKTOP |
                             USERRIGHT_TRANSMIT_DESKTOPINPUT |
-                            USERRIGHT_TRANSMIT_MEDIAFILE,
-                               
+                            USERRIGHT_TRANSMIT_MEDIAFILE |
+                            USERRIGHT_TEXTMESSAGE_USER,
```

Comment:
Shouldn't be also a "USERRIGHT_TEXTMESSAGE_CHANNEL" here?

Reply:
Yes, thanks!